modules/pubmaticBidAdapter.js

```diff
@@ -576,6 +578,26 @@ function _addPMPDealsInImpression(impObj, bid) {
   }
 }
 
+function _addDealCustomTargetings(imp, bid) {
```

Comment:
`_handleDealCustomTargetings` used to throw a warning if dctr param is not a `string`
Better to restore the warning

Reply:
Done